src/RangeSlider.webmodeler.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, createElement } from "react";
+import { RangeSlider, RangeSliderProps } from "./components/RangeSlider";
+import RangeSliderContainer, { RangeSliderContainerProps } from "./components/RangeSliderContainer";
+
+import * as css from "./ui/RangeSlider.scss";
+import * as rcsliderCss from "rc-slider/dist/rc-slider.css";
+
+// tslint:disable-next-line:class-name
+export class preview extends Component<RangeSliderContainerProps, {}> {
+    private warnings: string;
+
+    componentWillMount() {
+        this.addPreviewStyle(css, "widget-range-slider-preview-style");
+        this.addPreviewStyle(rcsliderCss, "widget-rcslider-preview-style");
+    }
+
+    render() {
+        this.warnings = RangeSliderContainer.validateSettings({
```

Comment:
Noting is done with the warning.

Please output the in an Alert component, but also render you widget as good as it gets.